### DISCUSSION TITLE
pytest_runtest_makereport hook doesn't seem to work

### DISCUSSION THREAD

**Discussion by hndang** [pytest-dev/pytest-html#661] [Q&A]
I can't seem to get this working.
From debugging it is called and the report obj is changed but the output html is not affected. I just copied the code in the documentation or just try with `report.description`

`
Packages	{"pluggy": "1.0.0", "pytest": "7.2.2"} 
Python	3.9.2
`

**Comment by BeyondEvil** [comment]
Could you provide a minimal reproducible example?

Also, please provide as much information about your environment as possible (OS, pip freeze etc.)

Thank you!
